src/main/java/com/blibli/future/model/Shift.java

```diff
@@ -2,22 +2,34 @@
 
 import javax.persistence.Entity;
 import javax.persistence.Id;
-import java.sql.Timestamp;
+import java.time.DayOfWeek;
+import java.time.LocalTime;
 
 @Entity
 public class Shift {
 	@Id
 	private String id;
```

Comment:
If we are using an Id, isn't it better to make it updated automatically? like `EmployeeAbsencePermit`?
```
@GeneratedValue(generator = "uuid")
@GenericGenerator(name = "uuid", strategy = "uuid2")
```